### DISCUSSION TITLE
Could I get a Changelog?

### DISCUSSION THREAD

**Discussion by Ypot** [orgzly-revived/orgzly-android-revived#17] [Q&A] [answered]
Hi

I would like to know the new features of the released APK, but the readme is not updated and I can't find a changelog.

Best regards

**Comment by amberin** [comment, marked as answer]
There is now a changelog on the page of the new beta release: https://github.com/orgzly-revived/orgzly-android-revived/releases/tag/v1.8.11-beta.2.

The "what's new" dialog in the app contains the same list of changes.
